pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -339,7 +339,8 @@ func (plugin *kubenetNetworkPlugin) TearDownPod(namespace string, name string, i
 	}
 
 	// no cached CIDR is Ok during teardown
-	if cidr, ok := plugin.podCIDRs[id]; ok {
+	cidr, ok := plugin.podCIDRs[id]
```

Comment:
uh....I just saw that `ok` is used below. Can we rename ok to `hasCIDR` or something more meaningful since it's not just used immediately?